### PR TITLE
Changes Declaration of War to always be possible.

### PR DESCRIPTION
**Pull request by GuyonBroadway**
:cl: GuyonBroadway
tweak: Nuke ops can now declare war despite the population and gain an amount of bonus tc based on how many players are on the server.
/:cl:

[why]:Nuke ops need some love. The declaration of war change alone should make for some more fun situations.

### CODE REVIEW

**Comment by Arzion5** [comment]
4 nuke ops on lowpop wouldn't be good though. Either way +1

**Comment by GuyonBroadway** [comment]
shoulda clarified I was taking our average pop of 20-35(ish) into account.

**Review by ghost — DISMISSED** [review]
i guess the code is ok

**Comment by ghost** [comment]
i'm a bit scared to merge this since i worry lowpop nukeops will get a huge boon in TC with their bigger squad when they declare war

like, i don't mind removing the declare war limit by itself
and i don't mind increasing the numbers at lowpop (with perhaps a small TC increase since more guys?)
but like, if we do the two together it might be too powerful at lowpop yeah?

**Comment by ghost** [comment]
make it require minimum 25 pop for war and i'll be ok

**Comment by GuyonBroadway** [comment]
Well I could adjust the nukie scaling but currently nukies are in a real bad spot, they get waaay less tc and members than they used to. As of now they get 6tc per player on the server so at around 40 pop they get 200tc like they used to, remember that declare war is a double edged sword that means you have about five min between you landing your shuttle and the emergency shuttle leaving assuming the station call it right away.

**Comment by PopNotes** [comment]
How about one nukie per ten players

**Comment by GuyonBroadway** [comment]
That's what we used to have and due to how it worked you tended to get two to three nukies at the 30 pop region.

**Comment by ghost** [comment]
i just think a bigger squad as such a lower number with way more guns while they do have to declare war, it is a matter of understaffed station against 3 syndies with one shotting guns

like, i mean we could try this! but it i do worry this will be excessive since you're changing 2 variables at once as a means to address a problem

**Comment by GuyonBroadway** [comment]
One other thing based on codediving, I think Jamie flipped nuke ops back to thier old 30 pop limit but I can't be sure.

This could do with a week on testmerge for sure.

**Comment by 2Dollarbill** [comment]
I'm a bit iffy on this, I think declaring war should be reserved for larger populations. However on lower populations I'm not sure if they really 1 - 2 people really need the extra TC.

**Comment by GuyonBroadway** [comment]
Remeber, as far as I can see, nukies was changed BACK so its not a lowpop game-mode anymore. 

if I am wrong I'll add a check but otherwise remember nukies has a 30 pop requirement.

**Comment by GuyonBroadway** [comment]
build this

**Comment by ghost** [comment]
Build started sha1 is merged.

File: hippiestation/code/game/gamemodes/nuclear/nuclear.dm [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/datum/game_mode/nuclear/pre_setup()
+	var/n_players = num_players()
+	var/n_agents = min(round(n_players / 7, 1), agents_possible) //One nukie per 7 players, that means at around 30 plyers we should see 3-4 nukies.
```
Comment (tortellinitony) [inline hippiestation/code/game/gamemodes/nuclear/nuclear.dm]: Multiply by the inverse of seven instead of dividing

Comment (tortellinitony) [reply hippiestation/code/game/gamemodes/nuclear/nuclear.dm]: also you don't need ,1

**Review by ghost — DISMISSED** [review]
Code seems fine

**Comment by j-mie** [comment]
Don't modify the scaling in this PR, I wanna see this without

**Comment by GuyonBroadway** [comment]
Can do.

**Comment by GuyonBroadway** [comment]
If jenkins says good then this is now good.

**Review by j-mie — APPROVED** [review]

**Pull request merged**